docs/04-logging.md

```diff
@@ -28,7 +28,7 @@ class Exp1(SimpleExperiment):
     return e
```

Comment:
I was quite puzzled about that myself :). We should just remove it or have it return a meaningful value